### ISSUE TITLE
index.js stops at "failed to find protobufs"?

### ISSUE THREAD

**Issue by VoiD199** [BeepIsla/csgo-commend-bot#210]
I might be dumb but it doesn't work. Here's the screenshot:
![prot](https://user-images.githubusercontent.com/55344042/72349225-112a9400-3717-11ea-8ae3-9c79e716ada9.PNG)

**Comment by xSuspect** [comment]
download this https://github.com/SteamDatabase/Protobufs/tree/fb83d09371b3911d489925e4d1100d7153c50c25

**Comment by BeepIsla** [comment]
Oh yeah I wanted to fix that a few days ago but forgot.

For now download what xSuspect sent and extract it into the protobufs folder, so it looks like this
```
<root>/index.js
<root>/config.json
<root>/protobufs
<root>/protobufs/steam
<root>/protobufs/csgo
```

**Comment by BeepIsla** [comment]
Fixed in ee06ff01e395ba2ad43df57e1ade65aeeff6050f, it should now work automatically again. Sorry for the inconvenience, I don't know why this randomly stopped working.
